ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -237,6 +237,11 @@ spec:
     - name: content-hash
       runAfter:
         - read-config
+      when:
```

Comment:
This can't be skipped using a when because the result of this task is used in other places.